index.bs

```diff
@@ -644,15 +647,15 @@ partial interface HTMLIFrameElement {
     |document|'s permissions policy.</p>
     <p>To get the <a>observable policy</a> for an Element |node|, run the
     following steps:</p>
-        1. Let |inherited policy| be a new ordered map.
-        2. Let |declared policy| be a new ordered map.
-        3. For each <a>supported feature</a> |feature|:
+        1. Let |inherited policy| be a new [=ordered map=].
```

Comment:
The inconsistency is mine, originally, but this algorithm uses "a new ordered map", while others use "an empty ordered map" to mean the same thing. Should we commit to one or the other?

(Or feel free to ignore; this is a pre-existing issue, and you don't have to clean up *everything* :) )

Reply:
(I'm not yet sure how I feel about the syntax, but we can also use `«[ ]»` as literal syntax for an empty map.)

Reply:
No strong opinions myself. I've updated this to use the `an empty [=ordered map=]` syntax you recommend for consistency.